From: Leo Singer <leo.singer@ligo.org>
Last-Update: 2018-01-25
Subject: Skip test that only works under tox
Forwarded: not-needed

---
 tests/test_pytest_mpl.py | 1 +
 1 file changed, 1 insertion(+)

--- pytest-mpl.orig/tests/test_pytest_mpl.py
+++ pytest-mpl/tests/test_pytest_mpl.py
@@ -146,6 +146,7 @@
 """
 
 
+@pytest.mark.skip(reason='works only under tox')
 def test_output_dir(tmp_path):
     test_file = tmp_path / "test.py"
     test_file.write_text(TEST_OUTPUT_DIR)
