From: Fukui Daichi <a.dog.will.talk@akane.waseda.jp>
Last-Update: 2022-10-23
Subject: Add mpl_image_compare to markers
Forwarded: not-needed

---
 setup.cfg | 1 +
 1 file changed, 1 insertion(+)

diff --git a/setup.cfg b/setup.cfg
index b952567..681aee4 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -43,6 +43,7 @@ test =
 [tool:pytest]
 testpaths = "tests"
 markers = 
+	mpl_image_compare
 	image: run test during image comparison only mode.
 	hash: run test during hash comparison only mode.
 filterwarnings = 
