From: Fukui Daichi <a.dog.will.talk@akane.waseda.jp>
Last-Update: 2022-10-23
Subject: Add mpl_image_compare to markers
Forwarded: not-needed

---
 setup.cfg | 1 +
 1 file changed, 1 insertion(+)

--- pytest-mpl.orig/tests/subtests/subtest/pytest.ini
+++ pytest-mpl/tests/subtests/subtest/pytest.ini
@@ -1,5 +1,6 @@
 [pytest]
 markers =
+    mpl_image_compare
     image: run test during image comparison only mode.
     hash: run test during hash comparison only mode.
 filterwarnings =
--- pytest-mpl.orig/setup.cfg
+++ pytest-mpl/setup.cfg
@@ -53,6 +53,8 @@
 	ignore:distutils Version classes are deprecated
 	ignore:the imp module is deprecated in favour of importlib
 	ignore:The NumPy module was reloaded
+markers =
+	mpl_image_compare
 
 [flake8]
 max-line-length = 120
