From: Ole Streicher <olebole@debian.org>
Date: Fri, 22 Dec 2023 15:35:06 +0100
Subject: Ignore deprecation warnings in pytest

Closes: #1058425
---
 setup.cfg | 1 +
 1 file changed, 1 insertion(+)

--- pytest-mpl.orig/setup.cfg
+++ pytest-mpl/setup.cfg
@@ -50,6 +50,7 @@
 norecursedirs = tests/subtests/subtest
 filterwarnings = 
 	error
+	ignore::DeprecationWarning
 	ignore:distutils Version classes are deprecated
 	ignore:the imp module is deprecated in favour of importlib
 	ignore:The NumPy module was reloaded
